web/src/pages/apd/activities/cost-allocation/CostAllocateFFPQuarterly.js

```diff
@@ -148,17 +144,20 @@ const CostAllocateFFPQuarterly = ({
                   <Controller
                     control={control}
                     name={`[${q}].inHouse.percent`}
-                    render={({ field: { onBlur, ...props } }) => (
+                    value={quarterlyFFP[year][q].inHouse.percent}
+                    render={({ field: { onBlur, value, ...props } }) => (
                       <PercentField
                         {...props}
                         className="budget-table--input-holder"
                         fieldClassName="budget-table--input__number"
                         label={`federal share for ffy ${year}, quarter ${q}, state`}
                         labelClassName="sr-only"
-                        onChange={setInHouse(q)}
+                        onChange={({ target: { value } }) => {
+                          setInHouse(q)(value);
+                        }}
                         onBlur={onBlur}
                         round
-                        value={quarterlyFFP[year][q].inHouse.percent * 100}
+                        value={(value * 100).toFixed(0)}
```

Comment:
if you remove the value from controller and render, then this would be like it was before but with toFixed(0)
```
value={(quarterlyFFP[year][q].inHouse.percent * 100).toFixed(0)}
```